### ISSUE TITLE
Using own Corpus for training and testing of ASR

### ISSUE THREAD

**Issue by Linus-win** [espnet/espnet#3664]
Hi,

I am trying to build a E2E ASR model using my own corpus. I have both a training set and a testing set of prepared corpus. (8K wave files and stm of each wave file) I read the instructions on "github.com/espnet/espnet/tree/master/eg2/TEMPLATE/" and followed the instructions to create a new asr1 task. 

1) Can I know where should I place my training and test set of prepared corpus? Do I placed them in ~/espnet/egs2/first_try/asr1/local/train/ and ~/espnet/egs2/first_try/local/test/?
2) Do I need to separate the wave file and the corresponding stm files? or can I put them together in the same folder?
3) To include my train and test corpus, I will need to edit the db.sh files. According the the TEMPLATE instructions I need to include 
YOUR_CORPUS=~/espnet/egs2/first_try/local/train/
How do I include specify the test corpus directory?

I think that's my questions so far. 
Thank you in advance for your time and input.

Linus
